src/api.jl

```diff
@@ -4670,18 +4681,24 @@ XPRSaddcuts,
 XPRSaddnames,
 XPRSdelrows.
 """
-function addrows(prob::XpressProblem, _srowtype::Vector{Cchar}, _drhs::Vector{Float64}, _drng, _mstart::Vector{Int}, _mclind::Vector{Int}, _dmatval::Vector{Float64})
+function addrows(prob::XpressProblem, _srowtype::Vector{Cchar}, _drhs::Vector{Float64}, _drng, _mstart::Vector{Cint}, _mrwind::Vector{Cint}, _dmatval::Vector{Float64})
```

Comment:
lets do _mstart::Vector{<:Integer} here?
i think its good for people using the api later on...

Reply:
The `Int` version is implemented below and calls the 64 bit version of the Xpress function.